.github/workflows/test.yml

```diff
@@ -141,3 +141,24 @@ jobs:
           path: ./
           exclude-dirs: complexity,formatting
           include-file-types: .c,.html
+  test-manifest-verifier:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup python environment
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.7.x'
```

Comment:
Is there any reason why we can't start using a later python version?

Reply:
Haven't tested with a more recent version, but can check with the latest.